apps/mysolarpvdivert.php

```diff
@@ -893,14 +964,18 @@ function load_bargraph(start,end) {
     if (has_wind) {
         wind_kwh_data = feed.getdataDMY(config.wind_kwh.value,start,end,"daily");
     }
-    
+    if (has_battery) {
+        var discharge_kwh_data = feed.getdataDMY(config.app.discharge_kwh.value,start,end,"daily");
```

Comment:
Do the same as `wind_kwh_data` and make it a `var` outside of the `if` and then set it inside? (I'm no JS guru, so I'm not actually sure on the difference here - scoping in JS always hugely confused me! But at least we should be consistent.)